site/content/docs/browser-checks/_index.md

```diff
@@ -25,135 +26,121 @@ Your critical interactions might be:
 
-await page.goto('https://app.checklyhq.com/login') // 2
+test('Login to Checkly', async ({ page }) => { // 2
+  await page.goto('https://app.checklyhq.com/login') // 3
 
-await page.type('input[type="email"]', 'john@example.com') // 3
-await page.type('input[type="password"]','mypassword')
-await page.click('.btn.btn-success.btn-block')
+  await page.locator('input[type="email"]').type('john@example.com') // 4
+  await page.locator('input[type="password"]').type('mypassword') // 4
+  await page.getByRole('button', { name: 'Log In' }).click() // 5
 
-await page.waitForSelector('.status-table') // 4
-await page.screenshot({ path: 'checkly_dashboard.png' })
-await browser.close()
+  const homeDashboardTable = page.getByTestId('home-dashboard-table')
+  await expect(homeDashboardTable).toBeVisible() // 6
+})
 ```
 {{< /tab >}}
 {{< /tabs >}}
 
-**1. Initial declarations:** We first import a framework (Puppeteer or Playwright) to control a browser.
-We also declare a `browser` and a `page` variable.
+**1. Initial declarations:** We first import the Playwright Test framework to control the browser.
+
+**2. Establish environment:** We use the `page` fixture without having to initialise a browser and create a new page manually. See the documentation on [Fixtures](https://playwright.dev/docs/api/class-fixtures) to learn more.
 
-**2. Initial navigation:** We use the `page.goto()` method to load the first page.
+**3. Initial navigation:** We use the `page.goto()` method to load the first page.
 
-**3. Fill out input fields and submit:** Using the `page.type()` and `page.click()` methods we enter our email address and
-password and click the login button. You would normally use environment variables here to keep sensitive data
+**4. Fill out input fields and submit:** Using the `page.type()` method we enter our email address and
+password. You would normally use environment variables here to keep sensitive data
 out of your scripts. See [Login scenarios and secrets](/docs/browser-checks/login-and-secrets/) for more info.
 
-**4. Wait for dashboard and close:** The expected behaviour is that the dashboard loads. In our case we can recognise this
-by the correct loading of the `.status-table` element in the page. For a visual reference, we also take a screenshot.
-We then close the browser. Our check is done.
+**5. Click Login button:** We use Playwright's `getByRole()` locator to find the login button and also `.click()` on it right away.
+
+**6. Wait for the dashboard:** The expected behaviour is that the dashboard loads. We assess this by checking whether the element with the test ID `home-dashboard-table` is visible. The `getByTestId()` method is looking for elements where the `data-testid` attribute matches the provided value.
+Playwright Test will automatically retry assertions until it succeeds or times out (default timeout is 5s). Moreover, when the test has finished, Playwright Test will automatically tear down the `page` fixture and clean up.
```

Comment:
Playwright Test will automatically retry the assertion until it succeeds or times out (default timeout is 5s)